crates/iota-sdk-ffi/src/types/graphql.rs

```diff
@@ -568,21 +770,21 @@ pub enum Direction {
 }
 
 #[derive(uniffi::Record)]
-pub struct ValidatorSet {
+pub struct ValidatorSetQuery {
```

Comment:
Do we actually need this type? It seems odd to even have it in the FFI

Reply:
@Thoralf-M

Reply:
Removed here https://github.com/iotaledger/iota-rust-sdk/pull/166/commits/abedc3805e2a12b2a7a898344f54a6f2c1546c20